### DISCUSSION TITLE
Elasticsearch 8.0.0 - ElastAlert Errors

### DISCUSSION THREAD

**Discussion by ar3diu** [jertel/elastalert2#727] [Q&A] [answered]
Hi,

I just upgraded a local cluster to Elasticsearch 8.0.0 and checked if the ElastAlert2 rules _(I'm using the docker image)_ still function. I have two flatline rules that detect when data sources stop sending logs to Elasticsearch. Both worked well and still do while using ES 7.x. 
Now, after upgraded to 8.x I get the following errors: 

```
WARNING:elasticsearch:GET https://localhost:9200/logs-*/_doc/_search?ignore_unavailable=true&size=0 [status:400 request:0.116s]
elastalert    
ERROR:elastalert:Error running terms query: RequestError(400, 'illegal_argument_exception', 'request [/logs-*/_doc/_search] contains unrecognized parameters: [ignore_unavailable], [size]')
```

Any workaround idea would be very helpful.

**Comment by nsano-rururu** [comment, marked as answer]
It doesn't work because it doesn't support it yet. It doesn't say that the document also supports it.

**Reply by nsano-rururu** [reply]
https://github.com/jertel/elastalert2/issues/92

**Reply by ar3diu** [reply]
Thank you very much for the clarifications!
